easybuild/easyconfigs/m/medaka/medaka-1.9.1-foss-2022a.eb

```diff
@@ -0,0 +1,67 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+# Updated to foss-2020b to use with artic tool
+# J. Sassmannshausen (GSTT/NHS UK)
+# Updated to 1.5.0
+# Jasper Grimm (UoY)
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.9.1'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [('Autotools', '20220317')]
+
+_minimap_ver = '2.24'
+dependencies = [
+    ('Python', '3.10.4'),  # includes cffi
+    # TensorFlow ~=2.8.0 required by medaka 1.8.1, see requirements.txt
```

Comment:
Not sure what to make of this comment since this is not 1.8.1 nor are we using TF 2.8.0

Reply:
this was probably copy-pasted from the previous version.
for this version the requirement is `tensorflow~=2.10.0`